app/models/associate_consultant.rb

```diff
@@ -36,7 +42,18 @@ def blank?
 
   private
 
+  def graduation_date_in_the_past?
+    if graduation_date.present?
+      Date.today > graduation_date
+    else
+      false
+    end
+  end
+
   def graduation_date
-    program_start_date + 24.months
+    reviews = self.reviews.where(review_type:'24-Month')
```

Comment:
Space missing after colon.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.